packages/widget/src/components/resource-amount-selector/resource-amount-selector.ts

```diff
@@ -103,6 +103,9 @@ export class ResourceAmountSelector extends BaseComponent {
   };
 
   _validateAmount(amount: string): boolean {
+    if (amount === '') {
```

Comment:
if I try to pass a number like `13.45` the `.` will be treated as empty string, and the amount will be reseted to zero. Firefox treats this differently. The dot repeats the actual value that is currently int the input, and then continues. This is why this block works on firefox but not on chromium based browsers

Reply:
I'm sorry, I don't follow. I just tried 13.45 on both chrome and firefox and it works perfectly

Reply:
my bad, I wasn't aware that the browser was taking into consideration the localization for the type of symbol you use for decimal places, in my case `,` and not `.`, so everything is fine here. First time using `,` for decimal places though 😄